docs/examples/datamover_advanced_voloptions.md

```diff
@@ -0,0 +1,349 @@
+
+- Install the VolSync operator using OLM.
+
+![Volsync_install](/docs/images/volsync_install.png)
+
+- We will be using VolSync's Restic option, hence configure a restic secret:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name>
+type: Opaque
+stringData:
+  # The repository encryption key
+  RESTIC_PASSWORD: my-secure-restic-password
+```
+
+<h1 align="center">Backup/Restore with CephFS ShallowCopy<a id="shallowcopy"></a></h1>
+
+- Have a stateful application running in a separate namespace with PVCs using 
+  CephFS as the provisioner
+
+- Have the default `storageClass` and `volumeSnapshotClass` as cephFS, as shown
+    in the [prerequisites](#pre-reqs)
+
+- Create a DPA similar to below:
+  - Add the restic secret name from the previous step to your DPA CR 
+    in `spec.features.dataMover.credentialName`. If this step is not completed 
+    then it will default to the secret name `dm-credential`.
+
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
+          prefix: velero
+        provider: aws
+  configuration:
+    restic:
+      enable: false
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+        - vsm
+  features:
+    dataMover:
+      credentialName: <restic-secret-name>
+      enable: true
+      volumeOptionsForStorageClasses:
+        ocs-storagecluster-cephfs:
+          sourceVolumeOptions:
+            accessMode: ReadOnlyMany
+            cacheAccessMode: ReadWriteMany
+            cacheStorageClassName: ocs-storagecluster-cephfs
+            moverSecurityContext: true
+            storageClassName: ocs-storagecluster-cephfs-shallow
+          destinationVolumeOptions:
+            moverSecurityContext: true
+```
+
+<hr style="height:1px;border:none;color:#333;">
+
+<h4> For Backup <a id="backup"></a></h4>
+
+- Create a backup CR:
+
+```
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns>
+spec:
+  includedNamespaces:
+  - <app-ns>
+  storageLocation: velero-sample-1
+```
+
+- Wait several minutes and check the VolumeSnapshotBackup CR status for `completed`: 
+
+`oc get vsb -n <app-ns>`
+
+`oc get vsb <vsb-name> -n <app-ns> -ojsonpath="{.status.phase}` 
+
+- There should now be a snapshot(s) in the object store that was given in the restic secret.
+- You can check for this snapshot in your targeted `backupStorageLocation` with a
+prefix of `/<OADP-namespace>`
+
+<h4> For Restore <a id="restore"></a></h4>
+
+- Make sure the application namespace is deleted, as well as any volumeSnapshotContents
```

Comment:
Is this just a precaution and there shouldn't be anything that cause faulty restores if VSCs are not cleared?